### DISCUSSION TITLE
Import CSV Data into Table Failure #7489

### DISCUSSION THREAD

**Discussion by kyletd** [Budibase/budibase#7491] [Help]
When i upload a csv into a table on a self hosted instance of budibase, The import is successful, but it does not appear in the table. If i export to csv from the same table and open the file, you can see all the data that was previously imported. Is there something i'm missing or doing incorrectly.

**Comment by R2bEEaton** [comment]
Hi @kyletd,

I just tried verifying this on Budibase v1.2.52 and wasn't able to reproduce your problem. I imported a CSV and the data showed up fine. What version are you on? Additionally, could you link your specific CSV file and/or app in order to verify if it's a problem with either of those?

Thanks!

**Reply by kyletd** [reply]
Hi @R2bEEaton , 

I am running hosted version 1.1.28  I'm not sure what you mean by linking, you want a screenshot?

**Reply by kyletd** [reply]
@R2bEEaton 

I've just updated and i do notice some changes. I will upload some more data and get back with you.

**Reply by R2bEEaton** [reply]
Sounds good. I meant that if it still doesn't work, send the specific CSV so I can test importing it on my own install.
